static/src/javascripts/lib/geolocation.js

```diff
@@ -49,7 +52,7 @@ const init = (): void => {
             ),
         });
         // Allows application logic to wait until geolocation is set.
-        mediator.emit(geoLocationSetEvent);
+        fireGeoLocationEstablishedEvent();
```

Comment:
not sure I see the benefit in making this a function - it only happens once and is a one-liner